workloads/fio/fio.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+
+# Log file will be on pod
+log = logging.getLogger(__name__)
+
+
+def setup(**kwargs):
+    """
+    setup fio workload
+
+    Args:
+        **kwargs (dict): fio setup configuration
+
+    Returns:
+        bool: True if setup succeeds else False
+    """
+    io_pod = kwargs['pod']
+    # For first cut doing simple fio install
+    # TODO: we can use fio container image
+    cmd = "yum -y install fio"
+    return io_pod.exec_cmd_on_pod(cmd)
+
+
+def run(**kwargs):
+    """
+    Run fio with params from kwargs
+
+    Args:
+        kwargs (dict): params for fio
+
+    Result:
+        result of command
+    """
+    io_pod = kwargs.pop('pod')
+    st_type = kwargs.pop('type')
+    path = kwargs.pop('path')
+
+    fio_cmd = "fio"
+    args = ""
+    for k, v in kwargs.items():
+        if k == 'filename':
+            if st_type == 'fs':
+                args = args + f" --{k}={path}/{v}"
+        args = args + f" --{k}={v}"
+    fio_cmd = fio_cmd + args
+
+    return io_pod.exec_cmd_on_pod(fio_cmd)
```

Comment:
should do some error checking before return, also lets print out the full fio log please

Reply:
In case if any command failures , that is already handle by exec_cmd_on_pod. So I am not doing any check here and passing the result as is. If there is exception then when user inspects the result object it raises that exception. Regarding logs Do you want it to be printed from inside exec_cmd_on_pod ?

Reply:
yeah it is better the output is printed inside fio run function, I was talking about fio failures which can be parsed but we can take this up later.

Reply:
filed: https://github.com/red-hat-storage/ocs-ci/issues/312